test/fixtures/slack_identifiers.yml

```diff
@@ -0,0 +1,11 @@
+deployer:
+  identifier: 'U1234567890'
```

Comment:
what I meant is the U1234567890 -> ident-deployer so the tests with hardcoded values are more readable